lib/sedna/core/joint_inference/joint_inference.py

```diff
@@ -106,6 +106,29 @@ def __init__(self, estimator=None, config=None):
             self.estimator.load(self.model_path)
         self.cloud = ModelClient(service_name=self.job_name,
                                  host=self.remote_ip, port=self.port)
+        hem = self.get_parameters("HEM_NAME")
+        hem_parameters = self.get_parameters("HEM_PARAMETERS")
+
+        try:
+            hem_parameters = json.loads(hem_parameters)
```

Comment:
1. the `hem_parameters` is needed by other feature(such as incremental learning), please place this loading code into JobBase class
2. hem_parameters is a list of key/value injected by GM, this should be consistent